extensions/survey/src/survey-preference.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component } from "@k8slens/extensions";
+import React from "react";
+import { observer } from "mobx-react";
```

Comment:
How is this being imported if it is not listed as a dep? Also, are we not exporting this as part of `@k8slens`?

Reply:
https://github.com/lensapp/lens/blob/master/src/renderer/bootstrap.tsx#L26-L33

and:
```javascript
// webpack.config.js
...
externals: [
      {
        "@k8slens/extensions": "var global.LensExtensions",
        "react": "var global.React",
        "mobx": "var global.Mobx",
        "mobx-react": "var global.MobxReact"
      }
    ],
```